src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -27,7 +27,8 @@
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/filesystem": "^3.4|^4.0|^5.0",
         "symfony/finder": "^3.4|^4.0|^5.0",
-        "symfony/routing": "^4.4.12|^5.1.4"
+        "symfony/routing": "^4.4.12|^5.1.4",
+        "symfony/var-exporter": "^4.4|^5.0"
```

Comment:
can be removed actually: var-exporter is already a transitive dep via symfony/cache and that won't change

Reply:
Removed, thanks for the info!